internal/manager/config.go

```diff
@@ -144,7 +145,9 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.IntVar(&c.APIServerBurst, "apiserver-burst", 300, "The Kubernetes API RateLimiter maximum burst queries per second")
 	flagSet.StringVar(&c.MetricsAddr, "metrics-bind-address", fmt.Sprintf(":%v", MetricsPort), "The address the metric endpoint binds to.")
 	flagSet.StringVar(&c.ProbeAddr, "health-probe-bind-address", fmt.Sprintf(":%v", HealthzPort), "The address the probe endpoint binds to.")
-	flagSet.StringVar(&c.KongAdminURL, "kong-admin-url", "http://localhost:8001", `The Kong Admin URL to connect to in the format "protocol://address:port".`)
+	flagSet.StringVar(&c.KongAdminURL, "kong-admin-url", "http://localhost:8001",
```

Comment:
Did we consider the [`StringSlice` flag type](https://pkg.go.dev/github.com/spf13/pflag#StringSlice)?

Reply:
We have not. I wanted to keep the interface as clear as possible at the same time preserving what's already in place.

As I think of it: having multiple flags specified might make more sense when providing multiple entries.

Given this, we can then introduce a secondary flag for the service name to distinguish between the 2 configuration approaches.

Reply:
I've tested this and it seems that it works well with this setup:

One can either specify the CLI flag multiple times or specify the value in the env using command separation.